content/en/apps/concepts/interoperability.md

```diff
@@ -4,13 +4,35 @@ linkTitle: "Interoperability"
 weight: 8
 description: >
   Exchanging information between CHT Core and other systems
-keywords: interoperability integrations
+keywords: interoperability integrations fhir icd openhie openhim
 relatedContent: >
   apps/features/integrations
 ---
 
-*Interoperability* and *Integration* are often used interchangeably but mean different things. [This article](https://www.linkedin.com/pulse/lets-talk-interoperability-vs-integration-mike-fleck) offers a useful delineation between the two.
+# Introduction
 
-With **interoperability**, systems work together, even if they weren’t designed specifically to work together, because standard communication protocol techniques are applied. Two (or more) systems that weren't designed to be interoperable require **integration**.
+Interoperability is the ability of health information systems to work together, even if they weren't specifically designed to work together. With interoperability, patient information can be seen, exchanged, and used across different platforms. This is different from _integration_ which requires custom development to connect two specific systems together. 
+
+Interoperability is the best practice for health systems because it allows information from one system to be shared with one or more other systems for with no additional development.
```

Comment:
I couldn't find one in a few min of searching - but a canonical "why interop is needed for CHIS success" link here would be good.  Not required!

Reply:
It's a little scary but I couldn't find one either. I'm going to leave it for now and keep iterating on this when I'm back...